docs/aca/09-aca-autoscale-keda/index.md

```diff
@@ -51,26 +54,25 @@ triggers:
 
 Let's review the parameters:
```

Comment:
In order to keep this consistent with rest of the page, How about we move details about what yaml file is doing to something like how we have done here https://azure.github.io/aca-dotnet-workshop/aca/04-aca-dapr-stateapi/#1-create-an-aca-dapr-component-file-for-state-store-management


![image](https://user-images.githubusercontent.com/8058415/229437554-5419e14f-d2b4-4f16-b536-3993d4d2d16e.png)